### PR TITLE
Improve README with installation notes

### PR DESCRIPTION
**Pull request by valentinarho**
In relation to issue #133 I propose a subsection of the README containing installation notes for problematic cases.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,13 +32,33 @@ Install the package:
 
     python setup.py install
 
+#### Istallation notes
```
Comment (amueller) [inline README.md]: Installation

File: README.md [hunk 2]
```diff
@@ -32,13 +32,33 @@ Install the package:
...
+
+##### Windows
+
 If you're having trouble with pip installation on windows, you can find a .whl file at:
 
 http://www.lfd.uci.edu/~gohlke/pythonlibs/#wordcloud
 
+##### OS different from Ubuntu
+
```
Comment (amueller) [inline README.md]: Actually that's no longer true and should be removed from the readme.

File: README.md [hunk 3]
```diff
@@ -32,13 +32,33 @@ Install the package:
...
 Note that if you are not on Ubuntu, you need to pass a ``font_path`` to the WordCloud object ([docs](http://amueller.github.io/word_cloud/generated/wordcloud.WordCloud.html#wordcloud.WordCloud)) to point to
 some existing font.
 
+##### Ubuntu
```
Comment (amueller) [inline README.md]: Thanks. The better solution would be to provide wheels but I don't have the time to set up the infrastructure for that right now. Feel free do to a PR for that.

File: README.md (hunk 1 above)
Comment (valentinarho) [inline README.md]: Typo fixed

File: README.md (hunk 2 above)
Comment (valentinarho) [inline README.md]: Removed

File: README.md (hunk 3 above)
Comment (valentinarho) [inline README.md]: I see. At this moment I don't have time too, but I queue it in my `future-tasks` list! :)

**Comment by amueller** [comment]
thanks :)

**Pull request merged**